lib/perl/Genome/InstrumentData/AlignmentResult.pm

```diff
@@ -1629,9 +1629,112 @@ sub verify_alignment_data {
 }
 
 sub alignment_bam_file_paths {
+    return glob(shift->output_dir . "/*.bam");
+}
+
+sub recreated_alignment_bam_file_paths {
+    my $self = shift;
+
+    my @bams = glob($self->output_dir . "/*.bam");
```

Comment:
This could call `alignment_bam_file_paths` instead of repeating the implementation.